designsafe/apps/api/datafiles/handlers/agave_handlers.py

```diff
@@ -0,0 +1,59 @@
+from designsafe.apps.api.datafiles.operations import agave_operations
```

Comment:
I'll have to think about that. The point of separating out the operations is to smooth the transition to Tapis V3- we just have to swap in a V3-compatible `operations.py`. It might be worth waiting until we know what the final form of these methods is going to be.